packages/components/psammead-bulleted-list/src/index.jsx

```diff
@@ -0,0 +1,33 @@
+import styled, { css } from 'styled-components';
+import { getBodyCopy } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { GEL_SPACING_TRPL } from '@bbc/gel-foundations/spacings';
+import Bullet from './bullet.svg';
+
+const dirBackground = css`
+  background: url("${encodeURIComponent(Bullet)}") no-repeat
+    ${({ dir }) => (dir === 'rtl' ? 'right' : 'left')} center;
+`;
+
+const dirMargin = css`
+  ${({ dir }) =>
+    dir === 'rtl'
+      ? `margin-right: -${GEL_SPACING_TRPL}`
+      : `margin-left: -${GEL_SPACING_TRPL}`}
+`;
+
+const BulletedList = styled.ul`
```

Comment:
You should probably include propTypes for `BulletedList` since you mentioned them in README

Reply:
Right